packages/storybook/stories/ControlBar/ControlBar.stories.tsx

```diff
@@ -83,12 +83,12 @@ const getDocs: () => JSX.Element = () => {
 
       <Heading>Example</Heading>
       <Description>
-        We recommend using our pre-defined buttons you can find [here](./?path=/docs/ui-components-controlbar-buttons)
-        or `DefaultButton`, a [Button](https://developer.microsoft.com/en-us/fluentui#/controls/web/button) component
-        from Fluent UI, as controls inside `ControlBar`. Pre-defined styles like `controlButtonStyles` or
-        `controlButtonLabelStyles` can also be imported and used as `DefaultButton` styles for easy styling.
-        `FluentThemeProvider` is needed around `ControlBar` to provide theming and icons. Learn more about theming
-        [here](./?path=/docs/theming--page).
+        We recommend using our pre-defined buttons you can start find
+        [here](./?path=/docs/ui-components-controlbar-buttons) or `DefaultButton`, a
```

Comment:
To prevent these breaking in the future can we use permalinks? https://storybook.js.org/docs/react/configure/sidebar-and-urls#permalinking-to-stories
It would be great to get even one example of this into the repo so any future links can be updated to use the permalink syntax

Reply:
(we should also have a work item to replace all internal links with permalinks)

Reply:
will look into this :)

Reply:
@JamesBurnside, I changed all the links I could to permalinks

Reply:
This will massively help prevent un-noticed broken links, out of interest which ones can't be permalinked?